base/irrationals.jl

```diff
@@ -203,7 +203,7 @@ big(::Type{<:AbstractIrrational}) = BigFloat
 function alignment(io::IO, x::AbstractIrrational)
     m = match(r"^(.*?)(=.*)$", sprint(show, x, context=io, sizehint=0))
     m === nothing ? (length(sprint(show, x, context=io, sizehint=0)), 0) :
-    (length(m.captures[1]), length(m.captures[2]))
+    (length(m.captures[1]::SubString), length(m.captures[2]::SubString))
```

Comment:
```suggestion
    (length(something(m.captures[1])), length(something(m.captures[2])))
```

Reply:
Is this different from the case in `base/path.jl`

Reply:
no, I thought it might be clearer to indicate either `::AbstractString` or `something` though in each case

Reply:
Alright, changed it to `something`. It's also more generic than the typeasserts, so that's nice.